PROTOCOL.md

```diff
@@ -679,11 +765,22 @@ Refer to the [appendix](#checkpoint-schema) for an example on the schema of the
  - Data files MUST be uniquely named and MUST NOT be overwritten. The reference implementation uses a GUID in the name to ensure this property.
 
 ## Append-only Tables
-When the table property `delta.appendOnly` is set to `true`:
-  - New log entries MUST NOT change or remove data from the table.
-  - New log entries may rearrange data (i.e. `add` and `remove` actions where `dataChange=false`).
+Enablement:
+ - If the table is on a Writer Version up to 6, the table property `delta.appendOnly` must be set to `true`.
+ - If the table is on Writer Version 7, the feature `appendOnly` must exist in the table `protocol`'s `writerFeatures`, and the table property `delta.appendOnly` must be set to `true`.
+
+When enabled:
+ - New log entries MUST NOT change or remove data from the table.
+ - New log entries may rearrange data (i.e. `add` and `remove` actions where `dataChange=false`).
+
+To remove the append-only restriction, the table property `delta.appendOnly` must be set to `false`.
 
 ## Column Invariants
+Enablement:
+ - If the table is on a Writer Version up to 6, Column Invariants is enabled when column metadata contains required keys.
```

Comment:
I would simply say that it is always enabled. I.e., you always have to check the metadata.

Also, doesn't this feature have a _minimum_ protocol version where it was introduced?